media-sound/ncmpcpp/ncmpcpp-0.9.2-r4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="featureful ncurses based MPD client inspired by ncmpc"
+HOMEPAGE="https://ncmpcpp.rybczak.net/ https://github.com/ncmpcpp/ncmpcpp"
+SRC_URI="https://rybczak.net/ncmpcpp/stable/${P}.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
should be `GPL-2+`, see: https://github.com/ncmpcpp/ncmpcpp/blob/master/src/ncmpcpp.cpp#L5-L8